examples/run_ursula_with_rest_and_dht_but_no_mining.py

```diff
@@ -19,14 +19,19 @@
 
 DB_NAME = "non-mining-proxy-node"
 
-_URSULA = Ursula(dht_port=3501, rest_port=3601, ip_address="localhost", db_name=DB_NAME, federated_only=True)
+_URSULA = Ursula(dht_port=3501,
+                 rest_port=3601,
+                 rest_host="localhost",
+                 dht_host="localhost",
+                 db_name=DB_NAME,
+                 federated_only=True)
 _URSULA.dht_listen()
 
 CURVE = ec.SECP256R1
```

Comment:
Nice try lizard people.

Reply:
The lizard people are innocent. This should be renamed to `TLS_CURVE`, though.

Perhaps an upgrade to `SECP384R1` is possible? If not, leave it with P-256 for TLS compatibility.